src/Microsoft.ML.Data/Transforms/ConversionsExtensionsCatalog.cs

```diff
@@ -19,11 +19,16 @@ namespace Microsoft.ML
     public static class ConversionsExtensionsCatalog
     {
         /// <summary>
-        /// Hashes the values in the input column.
+        /// Create a <see cref="HashingEstimator"/>, which hashes the data from the column specified in <paramref name="inputColumnName"/>
+        /// to a new column: <paramref name="outputColumnName"/>.
         /// </summary>
         /// <param name="catalog">The conversion transform's catalog.</param>
-        /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.</param>
-        /// <param name="inputColumnName">Name of the column to transform. If set to <see langword="null"/>, the value of the <paramref name="outputColumnName"/> will be used as source.</param>
+        /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.
+        /// This column's data type will be a vector of <see cref="System.UInt32"/>, or a <see cref="System.UInt32"/> based on whether the input column data types
+        /// are vectors or scalars.</param>
+        /// <param name="inputColumnName">Name of the column whose data will be hashed.
+        /// If set to <see langword="null"/>, the value of the <paramref name="outputColumnName"/> will be used as source.
+        /// This estimator operates over text, numeric, boolean, <see cref="KeyDataViewType"/> or <see cref="DataViewRowId"/> data types. </param>
```

Comment:
>DataViewRowId [](start = 109, length = 13)

do we need to mention DataViewRowId? is it group id? we should ask Tom about a phrase description for it.

Reply:
I don't think it is group id, it is the idv row id, the Uint128

---
In reply to: [276842426](https://github.com/dotnet/machinelearning/pull/3394#discussion_r276842426) [](ancestors = 276842426)